plugin/src/main/cpp/include/export/export_plugin.h

```diff
@@ -40,7 +40,7 @@
 
 using namespace godot;
 
-static const char *PLUGIN_VERSION = "2.0.3-stable";
+static const char *PLUGIN_VERSION = "4.0.0-stable";
```

Comment:
This one too, but it's fine. It would also be nice to have some sort of fail-safe to prevent us forgetting to update this in the future, but no need to worry about that on this PR